test/dist/scss.test.js

```diff
@@ -29,12 +29,8 @@ test('if an asset-path function is defined, it is used to generate asset paths w
   const css = results.css.toString('utf-8');
 
   const [usaAlertWarning] = css.match(/\.usa-alert--warning:before\s?{[^}]+}/);
-  const [usaInputError] = css.match(
-    /\.usa-textarea\.usa-input--error:not\(\.usa-input--inline\)\s?{[^}]+}/,
-  );
```

Comment:
I removed this assertion because it seemed to be testing essentially the same thing as the preceding line, which is that images are rewritten using the `$image-path` variable.